gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryError.java

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
I mean I don't feel comfortable adding build requirements to gcloud-java-storage, especially if that means there's a chance I might do it wrong and break their build. I'd rather keep the damage to gcloud-java-nio for now. You're of course welcome to copy this someplace else later if you're comfortable.

But I wouldn't do it just yet, since maven seems to behave differently in the IDE and the command line, and for now I can only get one of the two to compile (not both).